script/model/model_server.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+"""
+This file contains the python model server implementation.
+
+The server should be stateless but with caching of models.
+The message format that the model-server expects should be kept consistent with Messenger class in
+the noisepage source code.
+The command format should be kept with ModelServerManager in the noisepage souce
+code.
+
+TODO(Ricky):
+- Encapsulate the modelserver in a class
+- Modelserver side of callbacks (once we are more clear on the actions)
+- Model invocation could be improved.
+
+"""
+
+from __future__ import annotations
+import zmq
```

Comment:
nit: arrange includes to be standard python include order? generally,

system packages before custom packages
alphabetical